### ISSUE TITLE
Can I change "thresholdInMs"?

### ISSUE THREAD

**Issue by radical-edo** [GoogleChrome/lighthouse#9218]
Hi,

you're doing great work here and I love it. But I've been wondering is there a way to pass the `thresholdInMs` via some options. The one I'm talking about it in the `filter()` function here https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/bootup-time.js#L152

I've been trying to find it in options and in code, but to no avail :( If someone could point me in the right direction that would be great!

**Comment by brendankenny** [comment]
Yes, there is, but it will require a custom config file. That value is from an audit option, which can be set in a config.

Here's an example changing the score thresholds for running in a custom simulated desktop environment:

https://github.com/GoogleChrome/lighthouse/blob/ebe7a22df3dcccf1ddefbddeae3aac9a3aeedc13/lighthouse-core/config/lr-desktop-config.js#L24-L36

If you share more about what your goal is, we could maybe give some more specific advice on using audit options like this.

**Comment by radical-edo** [comment]
Hm, I must be doing something wrong... can't make it work :/

Here are the options I pass

```js
        {
          port: chrome.port,
          onlyAudits: ['bootup-time', 'metrics'],
          disableDeviceEmulation: true,
          throttlingMethod: 'provided',
          audits: [
            {
              path: 'bootup-time',
              options: { thresholdInMs: 10 },
            },
            { path: 'metrics' },
          ],
        }
```

I've tried changing the `path: 'bootup-time'` to `metrics/bootup-time` but still no effect. Also I'm assuming that the `metrics` is referring to the audit "metrics", where one one would get most of the metrics in one neat place.


Oh, and what I'm trying to achieve is to use the `bootup-times` to track down some "slow" scripts find what's the problem and boost the loading time and overall performance of my site. There are a lot, really a lot of scripts floating around and just making the bundles smaller just won't cut it. And usually there is a lot of parts that are not used. While making this I want to know how much I've gained.
